app/move/fields/create.js

```diff
@@ -469,12 +468,12 @@ module.exports = {
     items: [
       {
         id: 'move_agreed',
-        value: true,
+        value: 'yes',
```

Comment:
I need to supply some explanation for this. In the model, this is described as a string, so the validation was not working. I could have changed it to boolean in the model, but we use all these as strings so far, so I have changed it here.

Reply:
When you model, do you mean the API model?

Reply:
This change also feels like a perfect thing that could be a separate commit, using the commit message to describe why it was changed so when looking back it's clear to understand why.

This GitHub comment will likely get lost during time and somebody may change it back to a boolean, not knowing why it was changed to a string.